article/RNH_AR_EG_SSTJ_EP_MMC-2016.md

```diff
@@ -120,23 +125,31 @@ $$F_{NLS} = \frac{1}{2} \sum_{m}^{i=1} \left
 - (1-f)\exp(-\sum_{j=1}^{7}W_{ij}\gamma_{j})\right ]^{2}$$ {#eq:3}
 
 Similarly to the original article [@Hoy2014-lk,], the WLS procedure is only used here
-to obtain the intial guess for the free water elimination parameters, which were then
-used to initialize the non-linear convergence procedure (see below).
+to obtain the initial guess for the free water elimination parameters, which were then
+used to initialize the non-linear solver (see below).
 
 **Non-Linear Least Square Solution (NLS)**. As suggested by Hoy and colleagues
-[@Hoy2014-lk,], the initial guess for the non-linear convergence
-procedure was set to the values estimated from the WLS approach. To improve the speed of
+[@Hoy2014-lk,], the initial guess for the non-linear solver 
+was set to the values estimated from the WLS approach. To improve the speed of
 computation, instead of using the modified Newton's algorithm proposed in the original article,
 the non-linear convergence was done using Scipy's wrapped modified Levenberg-Marquardt algorithm
-(function `scipy.optimize.leastsq` of Scipy http://scipy.org/). To constrain the
+(function `scipy.optimize.leastsq` of Scipy http://scipy.org/). 
+
+stuff here
+
+To constrain the
 model parameters to within a plausible range, the free water volume fraction $f$ was
-converted to $f_t = \arcsin (2f-1) + \pi / 2$. To compare the robustness of the
+converted to $f_t = \arcsin (2f-1) + \pi / 2$. 
+
+major stuff here
```

Comment:
That's the part I'll admit I never understood - you should probably add a reference to going from unconstrained to constrained optimization through a non linear function. I'll have to admit this is either in the category 'wow this is a clever trick' or 'is that even allowed?', and I have no idea which one it belongs to unfortunately.